commons/phone/phone_bluetooth_devices.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneBluetoothDevices",
+  "doc": "Phone Bluetooth devices info.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "pairedDevices", "type": "int", "doc": "Number of paired devices"},
+    {"name": "nearbyDevices", "type": "int", "doc": "Number of nearby devices"},
```

Comment:
perhaps make the type `["null", "int"]`, so it can be `null` whenever bluetooth is not enabled. Same for `pairedDevices`.

Reply:
Actually the plugin always turns Bluetooth on (and switches it off back if it wasn't active), so it should be always able to get the numbers.

Reply:
Done